extensions/LIZARD_OFFICIAL/2FA.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable */
```

Comment:
Using `/* eslint-disable */` for the whole script is generally not a good idea, it's required for the library you imported but you should add an `/* eslint-enable */` afterwards.